index.md

```diff
@@ -54,9 +54,14 @@ When you are more familiar with the game, you can try selectively discarding Lib
 Since you want to ensure the play of as many Liberal policies as possible, you should nominate the player you trust the most to be chancellor. If you have little trust for any other player, you might want to pick a chancellor who has not been on a government yet to gather information on as many players as possible. Be careful when nominating a chancellor after three Fascist policies have been played, as the Fascists will win the game if Hitler is succesfully nominated. Note that if Hitler is playing well, they may be one of the most trusted players in the game. As such, you need to be on the lookout for strange votes which might point to Fascists pushing for a Hitler nomination.
 
 #### As a Fascist
-This section is still to be completed.
+As a Fascist, you are one of the few players who has full knowledge of everyone's roles. However, you must be careful not to give too much information with your nominations. At the early stages of the game when few governments have gone through it is often best to stay undercover. One option is to not take sides and pick chancellors that the group considers reasonable. Sometimes this will allow you to put two Fascists into a government but don't worry too much if it doesn't happen. 
+
+After the placement of three Fascist policies you must be more aware of your nominations. At this stage watch out for Liberal players hunting for suspicious nominations. As a Fascist you must balance up your desire to nominate Hitler or a fellow Fascist with putting through a sub-optimal government. All it takes to spoil trust gained for Hitler in the early stages of the game is for a Fascist to be too eager to approve it. You must also gauge your own suspicion within the group at this stage. If some people are suspicious of you and you can instead try to cast suspicion on innocent Liberals. Bearing in mind what may make nominating Hitler suspicious you can do the same on a Liberal.
```

Comment:
You must also gauge the group's suspicion of you at this stage and you may try to shift it onto unsuspecting Liberals.